lib/public/views/lhcPeriods/overview/LhcPeriodsModel.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { Observable, RemoteData } from '/js/src/index.js';
+import { getRemoteDataSlice } from '../../../utilities/fetch/getRemoteDataSlice.js';
+import { PaginationModel } from '../../../components/Pagination/PaginationModel.js';
+
+/**
+ * Model representing handlers for lhc periods page
+ *
+ * @implements {OverviewModel}
+ */
+export class LhcPeriodsModel extends Observable {
+    /**
+     * The constructor of the Overview model object
+     * @param {Model} model Pass the model to access the defined functions
+     * @returns {Object} Constructs the Overview model
+     */
+    constructor(model) {
```

Comment:
Avoid passing model further if you don't need it!

Reply:
applied